mava/systems/ppo/ff_ippo.py

```diff
@@ -81,6 +81,8 @@ def _env_step(learner_state: LearnerState, _: Any) -> Tuple[LearnerState, PPOTra
             key, policy_key = jax.random.split(key)
             actor_policy = actor_apply_fn(params.actor_params, last_timestep.observation)
             value = critic_apply_fn(params.critic_params, last_timestep.observation)
+            # sample,log_prob= actor_policy._sample_and_log_prob(
+            # seed=seed, sample_shape=(num_actions,))
```

Comment:
Unfortunate that tfp doesn't have `sample_and_log_prob`

```suggestion
```

Reply:
Yeah we'll get some typing issues 😿